composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
     	"php": ">=5.5.0",
         "guzzlehttp/guzzle": "^6.1",
-        "psr/log": "1.0.0",
+        "psr/log": "~1.0",
```

Comment:
`~1.0` and `^1.0` are the same, but it would it would conveniente to have the same notation for all dependencies.